packages/reaction-email-templates/server/hooks.js

```diff
@@ -0,0 +1,297 @@
+/**
+ * ReactionEmailFromTemplate - Returns a rendered and translated
+ * email template.
+ * @param {String} template name of the template
+ * @param {String} lng language of the email text
+ * @param {String} data data context for rendering
+ * @param {String} fallbackLng language that is used if lng has no translations
+ * @returns {Object} returns rendered HTML mail
+ */
+ReactionEmailFromTemplate = function (template, lng, data, fallbackLng) {
+  check(template, String);
+  check(lng, String);
+  check(data, Match.Optional(Object));
+  check(fallbackLng, Match.Optional(String));
+
+  if (typeof(fallbackLng) === "undefined") {
+    fallbackLng = "en";
+  } else if (fallbackLng !== "en") {
+    // also add english as second fallback..
+    fallbackLng = [fallbackLng, "en"];
+  }
+
+  if (!Template[template]) {
+      ReactionCore.Log.fatal(`Template '${ template }' does not exist. ` +
+        `Did you register it in the approriate workflow step (register.js)?`);
+  }
+  Template[template].helpers({
+    '_': function(key){
```

Comment:
rather that we didn't have helpers/methods named `_`.   Easily confused with underscore library, and also isn't descriptive. Hard to search for as well. This is one of the things I did not like about the `tap-i18n` package. This isn't doing the same thing the current i18n helper does?